tests/go/less_inherits.TODO

```diff
@@ -0,0 +1,3 @@
+type struct A {
```

Comment:
Go does not really have inheritance like the other languages. It does not fit in the less_xxx category.
However having dots for fields is a nice test to have, so maybe rename to
dots_struct_fields.TODO

Reply:
Ok addressed

Reply:
Ok addressed